test/hotspot/jtreg/compiler/splitif/TestSplitDivisionThroughPhi.java

```diff
@@ -74,6 +74,8 @@ public class TestSplitDivisionThroughPhi {
 
 
     public static void main(String[] strArr) {
+        Integer.divideUnsigned(2, 3);
+        Long.divideUnsigned(2, 3);
```

Comment:
Maybe add a comment here:
```suggestion
        // Make sure classes are loaded when compiling with -Xcomp
        Integer.divideUnsigned(2, 3);
        Long.divideUnsigned(2, 3);
```